.github/workflows/schemas.yml

```diff
@@ -0,0 +1,88 @@
+name: Update schema data
```

Comment:
Thoughts on including this as part of one of the other "see if this file changes and if so commit/push it" workflows? There's a lot of duplicate code overlap between them.

Reply:
I think so too - I've made a mistake here anyway so have dropped this temporarily.